projects/mysadna2/src/app/app.component.scss

```diff
@@ -0,0 +1,16 @@
+.app-navigation {
+    float: right;
+    >a {
+        margin: 5px;
+        width: 83px;
+        height: 24px;
+        font-family: OpenSans;
+        font-size: 18px;
+        font-weight: bold;
+        font-style: normal;
```

Comment:
You probably get all the `normal` styles from zeplin. But it's redundant. They do nothing and you can remove it.
I think the reason why zeplin show it, to reset all not normal values from another classes, which might be applies to the same element. But we don't have such issue.

Reply:
Do you mean I can simply remove all 'px' instances?

Reply:
No. I mean you can remove this:
```css
font-style: normal;
font-stretch: normal;
line-height: normal;
letter-spacing: normal;
```

Reply:
Implemented and pushed commit.